extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -106,6 +124,14 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
       StateProperty.SUBSCRIPTIONS_DIALOG_UI_STATE,
       (showDialog) => this.onSubscriptionsDialogUiStateChange_(showDialog)
     );
+
+    const ampSubscriptionsEl =
+      this.element.parentElement.parentElement.querySelector(
```

Comment:
Chaining `.parentElement.parentElement` is fragile, do not do this.
Components cannot rely on the structure of other components. 
Components need to be as de-coupled as possible.

Reply:
Yeah I agree this is not ideal. Do you know how to solve null "document' issue in the unit test?

Reply:
My bad. I found that I omit the part that target at the specific window we want. Fixed now.